service/src/main/java/org/jboss/sbomer/service/feature/sbom/model/SbomGenerationRequest.java

```diff
@@ -92,7 +92,7 @@ public class SbomGenerationRequest extends PanacheEntityBase {
     @ToString.Exclude
     private JsonNode config;
 
-    @Column(name = "reason", nullable = true, updatable = true)
+    @Column(name = "reason", nullable = true, updatable = true, length = 1024)
```

Comment:
I would add  `@Type(type = "org.hibernate.type.TextType")` and do not bother with max size? Unless we then make sure we trim the reason to be max 1024 chars when we create it. Btw we already set "text" type in stage and prod, so no worries about the length already.

Reply:
Good catch, will adjust!